timemachine/fe/chiral_utils.py

```diff
@@ -342,3 +347,44 @@ def find_chiral_bonds(mol):
             chiral_bonds.add(tuple(sorted([match[0], match[1]])))
 
     return chiral_bonds
+
+
+def _find_flipped_torsions(
+    torsions_a: Mapping[FourTuple, float], torsions_b: Mapping[FourTuple, float], core: Sequence[int]
+) -> List[ChiralConflict]:
+    results = []
+    for (ia, ja, ka, la), sign_a in torsions_a.items():
+        idxs_b = core[ia], core[ja], core[ka], core[la]
+        try:
+            sign_b = torsions_b[idxs_b]
+            if sign_a != sign_b:
+                results.append(((ia, ja, ka, la), idxs_b))
+        except KeyError:
+            pass
+
+    return results
+
+
+def setup_find_flipped_planar_torsions(mol_a, mol_b):
+    def enumerate_planar_torsions(mol):
+        conf = get_romol_conf(mol)
+        graph = convert_to_nx(mol)
+        idxs = {canonicalize_bond(tuple(idxs)) for idxs in enumerate_simple_paths(graph, 4)}
+
+        planar_torsions = dict()
+        for i, j, k, l in idxs:
+            bond_type = mol.GetBondBetweenAtoms(j, k).GetBondType()
+            if bond_type == BondType.DOUBLE or bond_type == BondType.AROMATIC:
+                volume = torsion_volume(conf[i], conf[j], conf[k], conf[l])
+                planar_torsions[(i, j, k, l)] = np.sign(volume)
```

Comment:
Possible to also special case amide bonds?

``` python
from rdkit import Chem

def find_canonical_amide_bonds(mol):
    query = Chem.MolFromSmarts("[NX3]-[CX3](=[OX1])")
    res = mol.GetSubstructMatches(query)
    
    amide_bonds = []
    for i,j,k in res:
        bonds.append(sorted([i,j]))
    
    return amide_bonds
    
mol = Chem.MolFromSmiles("CN1CCC(CC1=O)NC(C)=O")
find_canonical_amide_bonds(mol)
```

Reply:
Updated to include torsions with amide central bonds in the check in https://github.com/proteneer/timemachine/pull/1229/commits/53ae67da5c6239ddab18a7ff6012843bccea890d